tests/doxygen/message-format.md

```diff
@@ -0,0 +1,697 @@
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ \               RDM_DUB_Response (variable size)                \
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+</pre>
+
+@param Discovery_Start the time from the end of the transmitted DUB frame to
+the start of the DUB response, in 10ths of a microsecond. Undefined unless
+RC_OK was returned.
+@param Discovery_End the time from the end of the transmitted DUB frame to
+the end of the DUB response, in 10ths of a microsecond. Undefined unless
+RC_OK was returned.
+@param RDM_DUB_Response The raw response, if any was received.
+@returns
+- @ref RC_OK if the frame was sent correctly and data was received.
+- @ref RC_BUFFER_FULL if the transmit buffer is full.
+- @ref RC_TX_ERROR if a transmit error occurred.
+- @ref RC_RDM_TIMEOUT if no response was received.
+
+## Transmit Broadcast RDM Get / Set {#message-commands-txrdmbroadcast}
+
+Sends a broadcast RDM Get / Set command. If the Broadcast Listen Delay is 0,
+the transceiver will then listen for a response. If a response is received,
+it will be returned in the response payload.
+
+### Request Payload {#message-commands-txrdmbroadcast-req}
+
+<pre>
+  0                   1                   2                   3
+  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ \                        RDM_Command                            \
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+</pre>
+
+@param RDM_Command The broadcast RDM Get / Set command, excluding the start
+code.
+
+### Response Payload {#message-commands-txrdmbroadcast-res}
+
+<pre>
+  0                   1                   2                   3
+  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ |          Break_Start           |           Break_End          |
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ |           Mark_End             | RDM_Response (variable size) \
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ \                 RDM_Response (variable size)                  \
+ +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+</pre>
+
+@param Break_Start the time from the end of the transmitted RDM frame to
+the start of the break, in 10ths of a microsecond. Undefined unless
+RC_RDM_BCAST_RESPONSE was returned.
+@param Break_End the time from the end of the transmitted RDM frame to
+the end of the break / start of the mark, in 10ths of a microsecond.
+Undefined unless RC_RDM_BCAST_RESPONSE was returned.
+@param Mark_End the time from the end of the transmitted RDM frame to
+the start of the mark, in 10ths of a microsecond. Undefined unless
+RC_RDM_BCAST_RESPONSE was returned.
+@param RDM_Response The RDM response, if any was received.
+@returns
```

Comment:
It depends on what the broadcast delay was. Either the response will be ignored or it'll cause an error to be returned